core/src/test/scala/unit/kafka/server/RequestQuotaTest.scala

```diff
@@ -63,7 +63,6 @@ class RequestQuotaTest extends BaseRequestTest {
     properties.put(KafkaConfig.OffsetsTopicReplicationFactorProp, "1")
     properties.put(KafkaConfig.OffsetsTopicPartitionsProp, "1")
     properties.put(KafkaConfig.GroupMinSessionTimeoutMsProp, "100")
-    properties.put(KafkaConfig.GroupInitialRebalanceDelayMsProp, "0")
```

Comment:
I would leave this in, since the tests rely on the delay being very small - just in case, the default was changed in future.